api/server/pcap.go

```diff
@@ -36,8 +38,36 @@ type PcapAPI struct {
 }
 
 // SendFlows implements the flow Sender interface
-func (p *PcapAPI) SendFlows(flows []*flow.Flow) {
-	if p.Storage != nil && len(flows) > 0 {
+func (p *PcapAPI) SendFlows(item interface{}) {
```

Comment:
an alternative option is to add along side `SendFlows(flow []*flow.flow)` also `SendFlowUpdates(update []*flow.FlowUpdate)`

Reply:
+1, we can use interfaces this is too costly. Please add a function to the Flow Sender interface. I would suggest to add (if possible) an option the the flow table in order to specify if we want to get "Full Flow update" or "Partial Flow update", so that NetFlow target can remain as it is and this one too.